yahtzee.py

```diff
@@ -1,7 +1,10 @@
 #import packages
 from random import randint
 import numpy as np
-import pygame as pg
+import pandas as pd
+
+#instruction
+
 
  
  #define the Dice class
```

Comment:
I think it's very impressive you're using classes. But for each function in the class I would add a small comment to explain what e.g. __init__ / roll / etc does. For you it's probably super intuitive but for a reader (or at least for me) it would help if you explain it